packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -21,6 +21,7 @@ import {
   findNodeByRouting,
   splitPathIntoSegmentsWithoutRouteParams,
   IServicesCount,
+  debounce
```

Comment:
**question:** looks like it is not used. Do we need it?
```suggestion
```

Reply:
You're right, I deleted it